test/e2e-test-application/src/app/project/dynamic/dynamic.component.html

```diff
@@ -94,8 +94,8 @@ <h1 class="fd-section__title">{{ nodeLabel }}</h1>
   </div>
 </section>
 
-<section class="fd-section" *ngIf="nodeParams && hasBack">
-  <div class="fd-layout-panel fd-has-margin-bottom-small">
+<section class="fd-section" *ngIf="nodeParams || hasBack">
```

Comment:
Isn't *ngIf="nodeParams* duplicated in line 97 and 98?

Reply:
Nope, since we don't want to show the section at all, if none of the given parameters are defined.